bigtable/google/cloud/bigtable/instance.py

```diff
@@ -161,9 +162,18 @@ def create(self):
                     operation.
         """
         parent = self._client.project_path
+        cluster_path = self._client._instance_admin_client.cluster_path(
+            self._client.project, self.instance_id, 'shared-perf-cluster'
+        )
+        location_path = self._client._instance_admin_client.location_path(
+            self._client.project, 'us-central1-f'
+        )
+        clusters = []
+        clusters.append(instance_pb2.Cluster(name=cluster_path,
+                                             location=location_path))
```

Comment:
It cannot possibly be correct to jam this hardwired cluster configuration into `Instance.create`:  the method should take additional parameters (e.g., `cluster_id` and `location_id`) and use them to construct the two paths, if they are passed.

The *system tests* should then pass in the currently-hardcoded values (maybe with an environment check?) when creating an instance.

Reply:
Fixed